apps/web/pages/api/auth/setup.ts

```diff
@@ -10,8 +10,10 @@ import slugify from "@calcom/lib/slugify";
 import prisma from "@calcom/prisma";
 
 const querySchema = z.object({
-  username: z.string().min(1),
-  full_name: z.string(),
+  username: z
+    .string()
+    .refine((val) => val.trim().length >= 1, { message: "Please enter at least one character" }),
+  full_name: z.string().min(3, "Please enter at least 3 characters"),
```

Comment:
As this is the API, the messages are just in English and no advanced checking needed as we have in the UI